code/modules/pda/chatroom.dm

```diff
@@ -1,16 +1,252 @@
-var/list/chatrooms = list()
+var/list/chatrooms = list(new /datum/chatroom("General Discussion"))
 
 /datum/chatroom
```

Comment:
That's the app's type, `/datum/chatroom` is actually the chatroom/channel people connect to.